syncd/MdioIpcServer.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <ctype.h>
+#include <errno.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/select.h>
+#include <sys/socket.h>
+#include <sys/un.h>
+#include <time.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <pthread.h>
+
+#include "VendorSai.h"
+
+extern "C" {
+#include <sai.h>
+#include <saiversion.h>
+}
+
+namespace syncd
+{
+    class MdioIpcServer
+    {
+        public:
+
+            MdioIpcServer(
+                    _In_ std::shared_ptr<sairedis::SaiInterface> vendorSai,
+                    _In_ int globalContext);
+
+            virtual ~MdioIpcServer();
+
+        public:
+
+            static void setSwitchId(
+                    _In_ sai_object_id_t switchRid);
+
+            int startMdioThread();
+
+            void stopMdioThread();
+
+        public:
+
+            static sai_object_id_t mdioSwitchId;
+
+            static bool syncdContext;
```

Comment:
move to priivate, also add prefix m_ since thoese are members, if you need those to be exposed add getters for them, also indicated switchId whether this is Rid or Vid  like m_switchRid;

Reply:
Changed according to your comment.